Assets/MRTK/Core/Interfaces/InputSystem/IPointerPreferences.cs

```diff
@@ -44,6 +44,9 @@ public interface IPointerPreferences
         /// We make gaze pointer unique because the internal 
         /// gaze pointer actually cannot be referenced from here 
         /// since it's an internal class.
+        /// 
+        /// This does not control the Gaze Provider's Behavior. Set CoreServices.InputSystem.GazeProvider.Enabled 
```

Comment:
This comment seems weird to me.

> This does not control the Gaze Provider's Behavior.

feels like it should not be true for something called `GazePointerBehavior` that's settable...